runtime/v2/shim/util.go

```diff
@@ -37,6 +37,29 @@ import (
 
 var runtimePaths sync.Map
 
+const abstractSocketPrefix = "\x00"
+
+type socket string
+
+func (s socket) isAbstract() bool {
+	return strings.HasPrefix(string(s), "unix://")
```

Comment:
Should this be checking for `abstractSocketPrefix` instead? The logic around L50 seems to be "no `unix://` means abstract".

Reply:
good catch, thanks!